src/legacy/core_plugins/metrics/index.js

```diff
@@ -21,14 +21,16 @@ import { resolve } from 'path';
 
 import fieldsRoutes from './server/routes/fields';
 import visDataRoutes from './server/routes/vis';
+import SearchStrategiesRegister from './server/lib/search_strategies/search_strategies_register';
 
 export default function (kibana) {
   return new kibana.Plugin({
+    id: 'metrics',
```

Comment:
Could we rename that to `tsvb`, since it's the name we usually use for the feature.

Reply:
@spalger I am always confused what the `id` here is doing. Will that also influence the prefix for settings for that plugin, or will that prefix still be determined by the package name?

Reply:
@timroes  Please see 
>    x-pack\plugins\rollup\server\lib\search_strategies\register_rollup_search_strategy.js

It's just a ID of plugin which we add into server.plugins object

renamed to tsvb. Thanks

Reply:
Good question, @spalger do you know? Because it's not working without adding the 'id' here

Reply:
@alexwizp @timroes I've found out that the `id` doesn't influence on requiring plugins, since they are included by plugins directory names

Reply:
resolved